report_account_move_settlement/views/res_company_views.xml

```diff
@@ -6,11 +6,24 @@
         <field name="model">res.company</field>
         <field name="arch" type="xml">
             <xpath expr="//page[@name='report_info']" position="inside">
-                <group>
-                    <field name="seller_settlement_report_title" />
-                    <field name="seller_settlement_report_debit_comment" />
-                    <field name="seller_settlement_report_credit_comment" />
-                    <field name="seller_settlement_report_settlement_comment" />
+                <group name="settlement_report" string="Settlement Report">
+                    <field name="report_title" />
+                    <field name="debit_comment" />
+                    <field name="credit_comment" />
+                    <field name="settlement_comment" />
+                    <field name="settlement_report_type_ids">
+                        <tree>
+                            <field name="name" />
+                            <field name="company_id" />
+                            <field name="report_title" />
+                            <field name="case_number" />
+                            <field name="company_address" />
+                            <field name="debit_comment" />
+                            <field name="credit_comment" />
+                            <field name="settlement_comment" />
+                            <field name="active" />
```

Comment:
Remove this field, or add `context="{'active_test': False}"` to settlement_report_type_ids.